src/jobflow_remote/jobs/jobcontroller.py

```diff
@@ -2518,6 +2533,76 @@ def build_indexes(
             for idx in flow_custom_indexes:
                 self.flows.create_index(idx, background=background)
 
+    def create_indexes(
+        self,
+        indexes: list[str | list],
+        collection: DbCollection = DbCollection.JOBS,
+        unique: bool = False,
+        background: bool = True,
+    ) -> None:
+        """
+        Build the selected indexes
+
+        Parameters
+        ----------
+        indexes
+            List of indexes to be added to the collection. Each element is passed
+            to pymongo's create_index, thus following those conventions.
+        collection
+            The collection where the index will be created.
+        unique
+
+        background
+            If True, the indexes should be created in the background.
+        """
+        coll = self.get_collection(collection)
+        for idx in indexes:
+            coll.create_index(idx, background=background, unique=unique)
+
+    def drop_indexes(
+        self,
+        indexes: list[str | list],
+        collection: DbCollection = DbCollection.JOBS,
+        unique: bool = False,
+        background: bool = True,
+    ) -> None:
+        """
+        Build the selected indexes
+
+        Parameters
+        ----------
+        indexes
+            List of indexes to be added to the collection. Each element is passed
+            to pymongo's create_index, thus following those conventions.
+        collection
+            The collection where the index will be created.
+        unique
+
+        background
+            If True, the indexes should be created in the background.
+        """
+        coll = self.get_collection(collection)
+        coll.create_indexes(indexes, background=background, unique=unique)
```

Comment:
This does not drop indexes right ? Docstring needs update and the method should be updated. Is this used somewhere btw ?

Reply:
I wanted to implement this, then I thought it was not worth, but forgot to remove the initial copy/paste.
I would remove unless you think it might be useful to have.
In any case these functions in `JobController` are more to expose the functionality to the CLI. A user that is expert enough to know what is an index and to get to this part of the code will likely be able to just run `job_controller.jobs.drop_index()`.

Reply:
Yup I think you can remove